net/DevExtreme.AspNet.Data/DataSourceLoadOptionsBase.cs

```diff
@@ -67,12 +67,21 @@ public class DataSourceLoadOptionsBase {
         /// </summary>
         public string[] Select { get; set; }
 
+        /// <summary>
+        /// An array of data fields that constrains the <see cref="Select" /> expression.
+        /// The applied select expression is the intersection of <see cref="PreSelect" /> and <see cref="Select" />.
+        /// </summary>
         public string[] PreSelect { get; set; }
 
+        /// <summary>
+        /// A flag that indicates whether the select expression must be executed by the LINQ provider.
+        /// If set to false, the select operation is performed in memory.
+        /// </summary>
         public bool? RemoteSelect { get; set; }
 
         /// <summary>
-        /// A flag that indicates whether grouping parameters must be translated into LINQ.
+        /// A flag that indicates whether grouping must be executed by the LINQ provider.
```

Comment:
A flag that indicates whether grouping must be executed by the LINQ provider. -> A flag that indicates whether the LINQ provider should execute grouping.